### PR TITLE
Merge deploy-website action into build.yml

### PR DESCRIPTION
**Pull request by Goooler**
Like #901 did.
- Bump setup-java action of deploy-website to v2.
- Make deploy-website depends on jvm.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -59,3 +59,44 @@ jobs:
...
+        uses: actions/checkout@v2
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Configure JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: 14
```
Comment (JakeWharton) [inline .github/workflows/build.yml:80]:
```suggestion
```

We can just make this happen in the publish job

**Review by JakeWharton — APPROVED** [review]

File: .editorconfig [hunk 2]
```diff
@@ -2,7 +2,6 @@ root = true
 
 [*]
 indent_size = 2
-end_of_line = lf
```
Comment (JakeWharton) [inline .editorconfig:5]: Why remove this?

Comment (Goooler) [reply .editorconfig:5]: Eol should be handled by `.gitattributes`.

Comment (JakeWharton) [reply .editorconfig:5]: I'm not sure an IDE would read that file, though. Although I'm not sure whether _any_ idea actually uses CRLF by default anymore.

Comment (Goooler) [reply .editorconfig:5]: I test it in windows, make a new `Test.kt` with CRLF by default.

**Pull request merged**